src/tutorials/media_library.md

```diff
@@ -0,0 +1,221 @@
+2. Set the **X Offset** property to `100`, and the **Y Offset** to `55`.
+3. Set the **Width** property to `265`, and the **Height** to `60`.
+4. Set the **Dock** and **Anchor** property **Top Left**.
+
+![!Title and Description](../img/MediaLibraryTutorial/create_title_desc.png){: .center loading="lazy" }
+
+### Add Buy Brush
+
+The buy brush will display a button in the item panel and have a UI Text object as a child that will let the player know how much the item costs to purchase.
+
+1. Select the **Buy** brush in **My Brushes** and add it to the **Hierarchy**.
+2. Rename the **UI Image** in the **Hierarchy** to `Buy`.
+3. In the **Properties** window, enable the **Retain Aspect Ratio** option.
+4. Set the **X Offset** property to `-16`.
+5. Set the **Width** property to `155` and **Height** to `55`.
+6. Set the **Dock** and **Anchor** property **Middle Right**.
+
+![!Buy](../img/MediaLibraryTutorial/create_buy.png){: .center loading="lazy" }
+
+#### Create Cost Text
+
+The Buy button has space where the cost of the item can be shown.
+
+1. Create a UI Text object called `Cost` that is a child of the **Buy** image.
+2. Set the **X Offset** property to `58`.
+3. Set the **Width** property to `85`, and the **Height** to `35`.
+4. Set the **Dock** and **Anchor** property **Middle Left**.
+5. Set the **Vertical Justification** property to **Center**.
+
+![!Cost](../img/MediaLibraryTutorial/cost_text.png){: .center loading="lazy" }
+
+### Add Coins Brush
+
+The Coins brush will create a UI image that gives you an area where the player's total coins can be displayed.
+
+1. Select the **Coins** brush in **My Brushes** and add it to the **Hierarchy**.
+2. Rename the **UI Image** in the **Hierarchy** to `Coins`.
+3. In the **Properties** window, enable the **Retain Aspect Ratio** option.
+4. Set the **Y Offset** property to `-40`.
+5. Set the **Width** property to `270` and **Height** to `78`.
+6. Set the **Dock** and **Anchor** property **Bottom Center**.
+
+![!Coins](../img/MediaLibraryTutorial/create_coins.png){: .center loading="lazy" }
+
+#### Create Total Coins Text
+
+The coin's image has an area on it where the player's total coins can go.
+
+1. Create a UI Text object called `Player Coins` that is a child of the **Player Coins** image.
+2. Set the **X Offset** property to `100` and the **Y Offset** to `6`.
+3. Set the **Width** property to `133`, and the **Height** to `46`.
+4. Set the **Dock** and **Anchor** property **Middle Left**.
+5. Set the **Vertical Justification** property to **Center**.
+
+![!Player Coins](../img/MediaLibraryTutorial/player_coins.png){: .center loading="lazy" }
+
+## Summary
+
+With the support for transparency, the Media Library allows you to create a custom UI experience for your players that can not be found in any other game. The Media Library makes managing your content very easy because it is tied to your account, so all the content is accessible in any of your projects.
```

Comment:
Maybe instead of "cannot be found" (we could all use Kenney assets, etc) try "that are unique to your game and theme"